spec/mongo/crypt/data_key_context_spec.rb

```diff
@@ -0,0 +1,98 @@
+require 'mongo'
+require 'support/lite_constraints'
+
+RSpec.configure do |config|
+  config.extend(LiteConstraints)
+end
```

Comment:
Also unneeded?

Reply:
This one is needed because I've only imported `mongo` and `lite_constraints`, since I don't really need anything else in the spec helpers for this test. I can always import at least `lite_spec_helper` if you'd prefer!

Reply:
I see, it's fine as is then